index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<!-- my web -->
+<body>
+  <main id="main">
+
+    <h1 id="title">Dr. Norman Borlaug</h1> 
+    
+    <div id="img-div"> <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg">
+    <div id="img-caption">"Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists" </div>
+    </div>
+  
+  <div id="tribute-info">
+  <h2>Here's a time line of Dr. Borlaug's life:</h2>
```

Comment:
- [ ] **[OPTIONAL]** You can consider using the add code GitHub feature when you write "div" to make it look like this `div`. This will help distinguish your tags from regular text and make it easier for others to understand. Hope this helps! 👌